File: 602HW1/fourargs.py
# Copyright 2017 Lijun Xiao ljxiao@bu.edu
import sys
l=len(sys.argv)
if l<=5:
	for i in range(1,l):
		print(sys.argv[i])
else:
	for i in range(1,5):
		print(sys.argv[i])
	for i in range(5,l):
		print(sys.argv[i],file=sys.stderr)

File: 602HW1/fourargs.cpp
// Copyright 2017 Lijun Xiao ljxiao@bu.edu
#include <iostream>
int main(int argumentcount, char **arguments){
	int i;
	for(i=1;i<=4;i++)
	std::cout<<arguments[i]<<std::endl;
    if(argumentcount>4){
    	for(i;i<=argumentcount;i++)
    		std::cerr<<arguments[i]<<std::endl;

    }

	
	
	return 0;
		
	
 }
 